src/coordinator/index.js

```diff
@@ -38,6 +38,18 @@ import { toCamelCase } from '../utils/objects';
  * @property {string} updatedAt ISO 8601 Extended Format date/time string
  */
 
+/**
+ * @typedef {Object} ContxtOrganizationUsers
```

Comment:
This can be removed as it's the endpoint returns the same object as `ContxtUser`